model_drafts/tiles/extra.js

```diff
@@ -0,0 +1,31 @@
+var Sequelize = require('sequelize');
+
+function Extra(db){
+  var Extra = db.define('Extra', {
+    type: Sequelize.STRING,
+    reference: Sequelize.UUID
+  });
+  return Extra;
+}
+
+function Laser(db){
+  var Laser = db.define('Laser', {
+    count: Sequelize.INTEGER, //1-3
+    entersfrom: Sequelize.STRING, //not sure if these are necessary. maybe for drawing and/or calculating when it hits an obstacle
+    exitsto: Sequelize.STRING
+  });
+  return Laser;
+}
+
+function Flag(db){
+  var Flag = db.define('Flag', {
+    number: Sequelize.INTEGER //1-8
```

Comment:
I don't think so. it will either be on a tile or not and the tile will be put into the board at a particular location.  
The other option I see would be to have flags completely separate from the tiles and as a separate kind of add on.